homeassistant/components/command_line/switch.py

```diff
@@ -49,6 +51,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     switches = []
 
     for object_id, device_config in devices.items():
+        icon_template = device_config.get(CONF_ICON_TEMPLATE)
```

Comment:
I don't see the advantage of using a temporary variable here.
```suggestion
```

Reply:
In that case all are variables, with similar code style.
Yet in this case you are mixing code style.

Reply:
right, I see what you mean. thanks for your review